src/indicators/algorithms/exponentialMovingAverage.js

```diff
@@ -0,0 +1,49 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.indicators.algorithms.exponentialMovingAverage = function() {
+
+        var days = 9,
+            value = fc.utilities.fn.identity;
+
+        function undefinedArrayOfLength(length) {
```

Comment:
slidingWindow makes the undefined value configurable. We should probably decide whether that makes sense or not, then be consistent here.